server/pages/index.js

```diff
@@ -119,7 +122,19 @@ function getCurrentCommitShortChecksum() {
 	}
 }
 
-function getDefaultContext( request ) {
+function getFileContentsSafely( filepath ) {
+	return new Promise( function( resolve ) {
+		fs.readFile( filepath, function( err, content ) {
```

Comment:
This could be much easier if you use `fs.readFileSync`. For a build tool there's no significant difference, and the code could stay mostly the same as it ws before.